src/components/assets/transfer/BackToAsset.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="container--back">
+    <router-link to="/assets">
```

Comment:
Sure but I don't know what `database-like script` looks like. Could you help elaborate on it a little more? Some examples would be helpful.

Reply:
Do you mean like [this](https://router.vuejs.org/guide/essentials/named-routes.html)? If so, I've tried but it did't work properly.

https://www.loom.com/share/365a7ec4936c4554a470463008174d39

```
const routes: RouteRecordRaw[] = [
  {
    path: '/assets/:network',
    name: 'Assets',
    component: AssetsPage,
}
```

```
    <router-link :to="{ name: 'Assets', params: { network: 'astar' } }">
      <span> {{ $t('assets.transferPage.backToAssets') }}</span>
    </router-link>
```

 I've also faced quite a few issues related to Vue-router sometimes, not sure if this is because we are using SSR.

Reply:
I just came up with an alternative solution, let me try it.

Reply:
I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/479/commits/5dcf8e15688d8df1c3f083530ca70a8efed9b24c) commit.

Path:
https://github.com/AstarNetwork/astar-apps/blob/09f01d11f8b9a7892d85517e1d40532136f4430f/src/router/routes.ts#L20